common/tasks/benchmark_test.go

```diff
@@ -96,7 +96,7 @@ func (n *noopTask) HandleErr(err error) error        { panic("implement me") }
 func (n *noopTask) IsRetryableError(err error) bool  { panic("implement me") }
 func (n *noopTask) RetryPolicy() backoff.RetryPolicy { panic("implement me") }
 func (n *noopTask) Ack()                             { n.Done() }
-func (n *noopTask) Nack()                            { panic("implement me") }
+func (n *noopTask) Nack(err error)                   { panic("implement me") }
```

Comment:
`ack & nack` vs `ack(err error)`
one should be enough?

Reply:
True. But looks a bit weird to me that Ack with error means the processing logic failed.